modelci/client.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
```

Comment:
should not be named as client, since we already have many clients for model profiling.

Reply:
does refactor client to runs looks fine?
From mlflow https://github.com/mlflow/mlflow/blob/master/mlflow/runs.py